.golangci.yml

```diff
@@ -187,6 +187,8 @@ linters-settings:
         disabled: true # TODO: Do not disable
       - name: unhandled-error
         arguments: ['bytes\.Buffer\.Write']
+      - name: unhandled-error
+        disabled: true
```

Comment:
there is a duplication of `unhandled-error`

Reply:
Fixed, thanks!